client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -131,23 +134,75 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
     }
 
     // GG_NEEDS_REVIEW: TODO: input validation and better error handling https://sim.amazon.com/issues/P39478724
+    /*
+    Output example:
+    Deployment ID: Deployment UUID
+    Created on: Time in UTC when the deployment was created (dd-MM-uuuu HH:mm:ss UTC)
+    Status: DEPLOYMENT_STATUS
+    Deployment Status Details: DETAILED_DEPLOYMENT_STATUS
+    Deployment Error Stack: [List, of, deployment, error, if, any] (null if successful deployment)
+    Deployment Error Types: [List, of, deployment, error, types, if, any] (null if successful deployment)
+    Deployment Failure Cause: Deployment failure cause in text (null if successful deployment)
+     */
     @CommandLine.Command(name = "status",
             description = "Retrieve the status of a specific deployment", mixinStandardHelpOptions = true,
             versionProvider = com.aws.greengrass.cli.module.VersionProvider.class)
     public int status(@CommandLine.Option(names = {"-i", "--deploymentId"}, paramLabel = "Deployment ID",
             required = true) String deploymentId) {
 
-        LocalDeployment status = nucleusAdapterIpc.getLocalDeploymentStatus(deploymentId);
-        System.out.printf("%s: %s%n", status.getDeploymentId(), status.getStatus());
+        LocalDeploymentStatus status = nucleusAdapterIpc.getLocalDeploymentStatus(deploymentId);
+        StringBuilder statusBuilder = new StringBuilder();
+        statusBuilder.append(String.format(
+                "Deployment ID: %s\n", status.getDeploymentId()));
+        statusBuilder.append(String.format(
+                "Created on: %s\n", status.getCreatedOn()));
+        statusBuilder.append(String.format("Status: %s\n", status.getStatus()));
+        DeploymentStatusDetails deploymentStatusDetails = status.getDeploymentStatusDetails();
+        if (deploymentStatusDetails != null) {
+            statusBuilder.append(String.format("Deployment Status Details: %s\n",
```

Comment:
I believe the key name for this is "Detailed Deployment Status". Deployment Status Details is the name for the entire map

Reply:
True, my bad, will edit this quick